spec/unit/type/file/content_spec.rb

```diff
@@ -227,6 +227,30 @@
           end
         end
       end
+
+      SAVED_TIME = Time.now
+      [:ctime, :mtime].each do |time_stat|
+        [["older", SAVED_TIME-1, false], ["same", SAVED_TIME, true], ["newer", SAVED_TIME+1, true]].each do
+          |compare, target_time, success|
```

Comment:
it would be good to also test what happens when ctime/mtime checksum types are used when a file is created and deleted.